WcaOnRails/app/views/static_pages/contact.html.erb

```diff
@@ -16,14 +16,15 @@
   <p>
     <strong><%= t('contact.board.name_html') %></strong> - <%= mail_to "board@worldcubeassociation.org" %><br />
     <%= t('contact.board.info_html') %><br />
-    <%= @board_members.order(:name).map(&:name).join(", ") %>
+    <%= t('contact.members') %>: <%= Team.find_by_friendly_id(Team::BOARD_FRIENDLY_ID).current_members.includes(:user).map(&:user).sort_by(&:name).map(&:name).join(", ") %>
```

Comment:
This list was thrown here without much context.

Reply:
The `map.sort.map` feels a bit silly to me. How about this instead?

```ruby
Team.board.current_members.includes(:user).map { |team_member| team_member.user.name }.sort.join(", ")
```